dhis-2/dhis-api/src/main/java/org/hisp/dhis/trackedentity/TrackedEntityQueryParams.java

```diff
@@ -148,9 +148,7 @@ public class TrackedEntityQueryParams {
   /** Tracked entity types to fetch. */
   private List<TrackedEntityType> trackedEntityTypes = Lists.newArrayList();
 
-  /** Selection mode for the specified organisation units, default is ACCESSIBLE. */
-  private OrganisationUnitSelectionMode organisationUnitMode =
-      OrganisationUnitSelectionMode.DESCENDANTS;
+  private OrganisationUnitSelectionMode orgUnitMode = OrganisationUnitSelectionMode.DESCENDANTS;
```

Comment:
I mentioned that to @enricocolasante as well. Its used widely across 9 modules (other teams as well) so we opted not too.

1. go with what I did
2. go with what I did + create a separate issue for renaming the enum and more fields 😅
3. We can name the request parameter and its representing field in RequestParams `orgUnitMode`. Keep all other fields as `orgUnitSelectionMode`.

I am fine with either but would love to avoid 2. 😂